TranslatorReasonerAPI.yaml

```diff
@@ -55,6 +56,19 @@ paths:
                     'biolink:Gene':
                       - biolink:directly_interacts_with
                       - biolink:decreases_activity_of
+  /knowledge_map:
```

Comment:
Re: naming, I'd prefer meta_graph or meta_knowledge_graph. That would align with the SmartAPI/BTE notion of a MetaKG. It's not obvious to me that the phrase "knowledge map" would be interpreted by a naive reader much differently from "knowledge graph".

Reply:
The results on the last vote on this are in the minutes from Dec 10. They were quite split although /knowledge_map was the winner with a slight majority. I would be fine with reconsidering and revoting on the endpoint name as we complete this for real.